pubspec.yaml

```diff
@@ -144,7 +144,10 @@ dependencies:
 
   package_info_plus: 1.4.1
 
-  receive_sharing_intent: 1.4.5
+  receive_sharing_intent:
+    git:
+      url: https://github.com/ManhNTX/receive_sharing_intent.git
```

Comment:
Describe more clearly what you changed the library in this PR.

Reply:
I change the app group id like below 
<img width="694" alt="Screen Shot 2022-08-03 at 06 40 29" src="https://user-images.githubusercontent.com/107173849/182494831-5b701a65-9610-4e94-88a5-42be776229bf.png">

Reply:
We have any way to do it in compile time?

Reply:
> Please address what you change for iOS controller in an `ADR`. And please don't forget to reply comments.

Done

Reply:
@hoangdat  done